src/main/java/core/basesyntax/model/Report.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import java.util.Map;
+
+public class Report {
+    private Map<String, Integer> fruitQuantityMap;
```

Comment:
Follow the encapsulation principle. The 'fruitQuantityMap' should likely be private and accessed through getter and setter methods, unless there's a specific design choice dictating otherwise.